data-builder/parse_augments.py

```diff
@@ -50,6 +55,11 @@ def parse_augments():
                 if augmentName:
                     option['name'] = augmentName
 
+                if 'name' in option and option['name'].startswith("Set Augment: "):
```

Comment:
This supports the Set Augments having an actual `set` property on their option, so when added to an item, the set bonus will work correctly.